docs/build/tutorials/platform/subnets/setup-dfk-node.md

```diff
@@ -0,0 +1,71 @@
+# Run a Node on DFK Subnet
+
+## Introduction
+
+We recently deployed the DFK subnet on Avalanche mainnet and successfully launched the [DFK chain](https://subnets.avax.network/defi-kingdoms/dfk-chain/explorer) on that subnet. This subnet is maintained by the [DeFi Kingdom](https://defikingdoms.com/)'s team and is based on Avalanche's [subnet-evm](https://github.com/ava-labs/subnet-evm).
```

Comment:
Wasn't us that deployed the subnet, but DeFi Kingdoms team. So instead of "We recently" might be more neutral to say 'DFK subnet was recently deployed..." Next sentence makes it clear who actually did it and owns it.